src/main/java/com/faforever/client/preferences/ChatPrefs.java

```diff
@@ -48,6 +56,10 @@ public ChatPrefs() {
     chatColorMode = new SimpleObjectProperty<>(CUSTOM);
     idleThreshold = new SimpleIntegerProperty(10);
     chatFormat = new SimpleObjectProperty<>(ChatFormat.COMPACT);
+    autoJoinChannels = new SimpleListProperty<>(FXCollections.observableArrayList());
+    String localeLanguage = Locale.getDefault().getLanguage();
+    Optional<OfficialLanguageChannel> languageChannel = OfficialLanguageChannel.getChannelName(localeLanguage);
```

Comment:
Skip the variable assignment, if it is unused and call ifPresent directly (but make a linebreak for the ifPresent).